kvserver/rangefeed: minor test helper refactor

wenyihu6 opened the pull request:
**kvserver/rangefeed: rename testStream.Events to GetAndClearEvents**

This patch renames testStream.Events to testStream.GetAndClearEvents for
better clarity. We also plan to introduce GetEvents() in a future commit.

Epic: none
Release note: none

----

**kvserver/rangefeed: change testRangefeedCounter.get to return int**

This patch changes testRangefeedCounter.get() is changed to return an int
instead of int32, making require.Equal() easier in tests.

Epic: none
Release note: none

wenyihu6 committed:
kvserver/rangefeed: rename testStream.Events to GetAndClearEvents

This patch renames testStream.Events to testStream.GetAndClearEvents for
better clarity. We also plan to introduce GetEvents() in a future commit.

Epic: none
Release note: none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/133685)
<!-- Reviewable:end -->

wenyihu6 committed:
kvserver/rangefeed: change testRangefeedCounter.get to return int

This patch changes testRangefeedCounter.get() is changed to return an int
instead of int32, making require.Equal() easier in tests.

Epic: none
Release note: none

wenyihu6 commented:
TFTR!

bors r=stevendanna
